tests/functional/api/test_flags.py

```diff
@@ -5,7 +5,7 @@ class TestPutFlag:
     def test_it_returns_http_204_if_user_allowed_to_flag_shared_annotation(
         self, app, annotation, user_with_token
     ):
-        user, token = user_with_token
+        _, token = user_with_token
```

Comment:
This is super weird. Why do we have a fixture where we get two bits but only want one? Seems pretty clear we'd like the token a lot?